### PR TITLE
Explain why issue is closed when not using an issue template

### PR DESCRIPTION
**Pull request by sampaiodiego**
## Proposed changes

Just add to the comment used to close the issue an explanation on what is the minimum required content an issue must have to not be closed. The new proposed comment is:

>  This issue was closed because it does not use our bug report issue template.
>
>  Please make sure to use it and fill it as much as you can so we can provide better and faster support.
>
>  The following sections **must not** be removed, or else the BOT will close it immediately again:
>    * Steps to reproduce
>    * Expected behavior
>    * Actual behavior
>    * Server Setup Information

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Improvement (non-breaking change which improves a current function)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Hotfix (a major bugfix that has to be merged asap)
- [x] Documentation Update (if none of the other choices apply)

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/RocketChat/Rocket.Chat/blob/develop/.github/CONTRIBUTING.md#contributing-to-rocketchat) doc
- [x] I have signed the [CLA](https://cla-assistant.io/RocketChat/Rocket.Chat)
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if applicable)
- [ ] I have added necessary documentation (if applicable)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: .github/issue-close-app.yml [hunk 1]
```diff
@@ -1,7 +1,17 @@
 # Config for Close Issue app: https://probot.github.io/apps/close-issue/
 
 # Comment that will be sent if an issue is judged to be closed
-comment: "This issue was closed because it does not use any of our issue templates. Please make sure to use one of the suggested templates."
+comment: |
+  This issue was closed because it does not use our bug report issue template.
+
+  Please make sure to use it and fill it as much as you can so we can provide better and fast support.
+
+  The issue **must** contain at least the following info:
```
Comment (Sing-Li) [inline .github/issue-close-app.yml:9]: All contributors facing this have deleted sections that don't apply.   Suggest ...

The following sections **must** not be removed, or else the BOT will close it immediately again.

Comment (sampaiodiego) [reply .github/issue-close-app.yml:9]: "Version of Rocket.Chat Server" in the list is not actually a section, but I change the item to be "Server Setup Information" instead, wdyt?

Comment (Sing-Li) [reply .github/issue-close-app.yml:9]: Sure, please do.    We need that entire section anyways!  Thanks.

Comment (sampaiodiego) [reply .github/issue-close-app.yml]: changed the text as you suggested =)

Comment (Sing-Li) [reply .github/issue-close-app.yml]: LGTM

File: .github/issue-close-app.yml [hunk 2]
```diff
@@ -1,14 +1,25 @@
 # Config for Close Issue app: https://probot.github.io/apps/close-issue/
 
 # Comment that will be sent if an issue is judged to be closed
-comment: "This issue was closed because it does not use any of our issue templates. Please make sure to use one of the suggested templates."
+comment: |
+  This issue was closed because it does not use our bug report issue template.
+
+  Please make sure to use it and fill it as much as you can so we can provide better and fast support.
```
Comment (Sing-Li) [inline .github/issue-close-app.yml:7]: ... better and faster ....

**Pull request merged**